documentation/SA1001.md

```diff
@@ -27,6 +27,7 @@ A comma should be followed by a single space, except in the following cases.
 
 * A comma may appear at the end of a line
 * A comma should not be followed by a space when used in an open generic type in a `typeof` expression
+* A comma is part of an alignment component. For example:`$"{x,3}"`
```

Comment:
I would prefer `a string interpolation alignment component`, instead of `an alignment component`